include/lsst/utils/Backtrace.h

```diff
@@ -22,8 +22,8 @@
  */
 
 
-#ifndef LSST_UTILS_BACKTRACE_H
-#define LSST_UTILS_BACKTRACE_H
+#ifndef LSST_CPPUTILS_BACKTRACE_H
+#define LSST_CPPUTILS_BACKTRACE_H
 
 namespace lsst {
 namespace utils {
```

Comment:
It should be possible to add:
```
namespace cpputils = utils;
```
inside the `namespace lsst { }` and then symlink each of these files to its equivalent in an `include/lsst/cpputils` tree to begin preparing for a full transition.

Reply:
so maybe we should relocate them and rename everything but leave the softlinks and namespace alias for the old names?

Reply:
Yes, if you're willing to go that far right now, sure.